terraform/microceph/README.md

```diff
@@ -0,0 +1,64 @@
+# Terraform module for MicroCeph
+
+This is a Terraform module facilitating the deployment of the `microceph` charm,
+using the [Terraform Juju provider](https://github.com/juju/terraform-provider-juju/).
+For more information,
+refer to the provider [documentation](https://registry.terraform.io/providers/juju/juju/latest/docs).
+
+> [!IMPORTANT]
+> This module requires a Juju K8s model to be available.
+> Refer to the [usage section](#usage) below for more details.
+
+> [!NOTE] This module temporarily lives here until the microceph charm provides its own.
+
+## Usage
+
+Users should ensure that Terraform is aware of the `juju_model` dependency of the charm module.
+
+To deploy this module with its needed dependency, you can run:
```

Comment:
should we mention to first run `terraform init`? and maybe plan, just to make sure everything is ok?

Reply:
That's plain terraform commands, I'd leave that out of this readme.